fulp_modules/antag_tips/html/changeling.html

```diff
@@ -0,0 +1,15 @@
+<html>
+	<center>
+	<h1>You are a Changeling!</h1>
+	<img src="changeling.gif"/>
+	<p>You are a Changeling, a shapeshifting alien assuming the form of a crewmember on Space Station 13.</p>
+	<p>The goal of many changelings is to acquire 5-7 DNA strains. To do this, it must take ANY human, living or dead, and acquire their DNA.</p>
+	<p><img src="sting_extract.png"/>Use the <b>DNA Extraction Sting</b> and sting a target to stealthily steal their DNA. This will count towards your objective.</p>
+	<p><img src="absorb.png"/>You can <b>absorb</b> humans to drain their DNA, husking them.</p>
+	<p><img src="emporium.gif"/>Use the <b>Cellular Emporium</b> to acquire special abilities which will help you achieve your objectives.</p>
+	<p>Absorbing a human will allow you to <b>readapt</b> and purchase different abilities.</p>
```

Comment:
you should say body instead, to prevent them from murdering random crewmates because "the tip said so"